pkg/cloudprovider/suite_test.go

```diff
@@ -1149,7 +1150,9 @@ var _ = Describe("CloudProvider", func() {
 					Tags: []ec2types.Tag{{Key: aws.String("Name"), Value: aws.String("test-subnet-2")}}},
 			}})
 			controller := status.NewController(env.Client, awsEnv.SubnetProvider, awsEnv.SecurityGroupProvider, awsEnv.AMIProvider, awsEnv.InstanceProfileProvider, awsEnv.LaunchTemplateProvider)
+			versionController := controllersversion.NewController(awsEnv.VersionProvider)
```

Comment:
I'm wondering if we should have a fake version provider that we use for most tests - needing to reconcile the version controller for anything that relies on the version being hydrated is awfully inconvenient. Another option would be to disregard my previous comment and call UpdateVersion from Get if the value hasn't been hydrated yet. I don't have a strong opinion either way, and I don't think we need to block on this.